### PR TITLE
Han1ya/create calendar component

### PR DESCRIPTION
**Pull request by han1ya**
added the date numbers to each day, shortened the day names to match google calendar, and centered the days. work was done in weeksGlance but will be transferred to calendar.js shortly

### CODE REVIEW

File: src/components/common/Calendar/Calendar.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import React, { useState } from "react";
+import {
+  Wrapper,
+  Event,
+  WeekCalendar,
```
Comment (Digital-Coder) [inline src/components/common/Calendar/Calendar.js:5]: WeekCalendar does not exist in StyledWeeksGlance

File: src/components/common/Calendar/Calendar.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        });
+        console.log(tempEventList);
+        setEventList(tempEventList);
+      })
+      .catch(err => {
+        console.error(err);
+      });
+  };
+
+  React.useEffect(componentDidMount, []);
```
Comment (Digital-Coder) [inline src/components/common/Calendar/Calendar.js:41]: React.useEffect(**componentDidMount**, []);
a bit confusing naming, I thought inside functional useEffect you used class based ComponentDidMount() lifecycle method, but here just data is being fetched on mount

**Review by NateD247 — APPROVED** [review]
Thank you for adding these common components and making the changes to the calendar!

**Pull request merged**